packages/components/src/components/form/Select/index.tsx

```diff
@@ -32,37 +34,39 @@ const selectStyle = (
         margin: 0,
         padding: '0 8px',
         color: isClean ? theme.TYPE_LIGHT_GREY : theme.TYPE_DARK_GREY,
-        fontSize: variables.NEUE_FONT_SIZE.SMALL,
-        fontWeight: variables.FONT_WEIGHT.MEDIUM,
+        fontSize: NEUE_FONT_SIZE.SMALL,
+        fontWeight: FONT_WEIGHT.MEDIUM,
         borderStyle: 'none',
         justifyContent: isClean ? 'flex-end' : 'flex-start',
+        position: 'static',
+        transform: 'none',
         '&:hover': {
             cursor: hideTextCursor || !isSearchable ? 'pointer' : 'text',
         },
     }),
     control: (base, { isDisabled, isFocused }) => {
-        let height = variant === 'small' ? '36px' : '48px';
+        let height = variant === 'small' ? 36 : 48;
         const borderColor = selectState ? getStateColor(selectState, theme) : theme.STROKE_GREY;
 
-        if (isClean) height = '22px';
+        if (isClean) height = 22;
 
         return {
             ...base,
             display: 'flex',
             alignItems: 'center',
-            fontSize: variables.FONT_SIZE.SMALL,
+            fontSize: FONT_SIZE.SMALL,
             height,
-            borderRadius: `${borderRadius}px`,
-            borderWidth: `${borderWidth}px`,
+            // borderRadius: 8,
```

Comment:
forgotten comments?

Reply:
Yesss, the first pull was a rough WIP :)